<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>CreateUpdatePage</class>
    <widget class="QWidget" name="CreateUpdatePage">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>362</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>0</width>
                <height>0</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: #fff</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
                <widget class="QFrame" name="frame">
                    <property name="styleSheet">
                        <string notr="true">color: black; font-size: 15px</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span
                                        style=&quot; font-size:11pt;&quot;&gt;Окно создания/обновления партнёра&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_12">
                                <item>
                                    <widget class="QLabel" name="Type">
                                        <property name="text">
                                            <string>Тип компании:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="verticalSpacer_12">
                                        <property name="orientation">
                                            <enum>Qt::Orientation::Vertical</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>20</width>
                                                <height>40</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="TypeInput"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_11">
                                <item>
                                    <widget class="QLabel" name="Name">
                                        <property name="text">
                                            <string>Название компании:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="verticalSpacer_11">
                                        <property name="orientation">
                                            <enum>Qt::Orientation::Vertical</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>20</width>
                                                <height>40</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="NameInput"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QLabel" name="Address">
                                        <property name="text">
                                            <string>Юр. адрес:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="verticalSpacer">
                                        <property name="orientation">
                                            <enum>Qt::Orientation::Vertical</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>20</width>
                                                <height>40</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="AddressInput"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_9">
                                <item>
                                    <widget class="QLabel" name="INN">
                                        <property name="text">
                                            <string>ИНН:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="verticalSpacer_9">
                                        <property name="orientation">
                                            <enum>Qt::Orientation::Vertical</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>20</width>
                                                <height>40</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="INNInput"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_10">
                                <item>
                                    <widget class="QLabel" name="BossName">
                                        <property name="text">
                                            <string>Имя директора:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="verticalSpacer_10">
                                        <property name="orientation">
                                            <enum>Qt::Orientation::Vertical</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>20</width>
                                                <height>40</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="BossNameInput"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_15">
                                <item>
                                    <widget class="QLabel" name="PhoneNumber">
                                        <property name="text">
                                            <string>Номер телефона:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="verticalSpacer_15">
                                        <property name="orientation">
                                            <enum>Qt::Orientation::Vertical</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>20</width>
                                                <height>40</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="PhoneNumberInput"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_14">
                                <item>
                                    <widget class="QLabel" name="Mail">
                                        <property name="text">
                                            <string>Эл. почта:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="verticalSpacer_14">
                                        <property name="orientation">
                                            <enum>Qt::Orientation::Vertical</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>20</width>
                                                <height>40</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="MailInput"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_13">
                                <item>
                                    <widget class="QLabel" name="Rank">
                                        <property name="text">
                                            <string>Рейтинг:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="verticalSpacer_13">
                                        <property name="orientation">
                                            <enum>Qt::Orientation::Vertical</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>20</width>
                                                <height>40</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="RankInput"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="QPushButton" name="CreateUpdateButton">
                                <property name="styleSheet">
                                    <string notr="true">border: 1px solid black;
                                        background-color: #67BA80
                                    </string>
                                </property>
                                <property name="text">
                                    <string>PushButton</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="pushButton">
                                <property name="styleSheet">
                                    <string notr="true">border: 1px solid black;
                                        background-color: #67BA80
                                    </string>
                                </property>
                                <property name="text">
                                    <string>BackButton</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
